common/src/main/scala/com/gu/media/youtube/MediaAtomYoutubeDescriptionHandler.scala

```diff
@@ -0,0 +1,23 @@
+package com.gu.media.youtube
+
+import com.gu.contentatom.thrift.atom.media.{MediaAtom => ThriftMediaAtom}
+import com.gu.media.model.MediaAtom
+import com.gu.media.util.MediaAtomImplicits
+import org.cvogt.play.json.Jsonx
+
+
+object MediaAtomYoutubeDescriptionHandler extends MediaAtomImplicits {
+
+  private implicit val mediaAtomFormat = Jsonx.formatCaseClass[MediaAtom]
+
+  def extractYoutubeDescriptionFrom(data: ThriftMediaAtom): Option[String] = {
```

Comment:
minor: would `getYoutubeDescription` be more descriptive here?

Reply:
done